packages/forklift-console-plugin/src/modules/Plans/PlansPage.tsx

```diff
@@ -142,8 +143,9 @@ const Page = ({
   return (
     <>
       {loadedDataIsEmpty && <EmptyStatePlans namespace={namespace} />}
+      {isLoading && <ConsoleLoadingDots isLoading={isLoading} />}
```

Comment:
``` ts
if (isLoading) {
  return <Loading>
}

if (isDataEmpty) {
  return <EmptyState>
}

return <Page>
```
WDYT ?

Reply:
The current way it's rendered is a leftover from first draft. I'll fix that.  The main feature here is to start the spinner with a delay.

Reply:
More logic is needed when trying to determine if the full page spinner should be shown.  

What happens when the queries are running to refresh/update the existing information?  This configuration will probably hide the table when that operation is ongoing.

Reply:
right. the dots should be shown only on initial load.